### PR TITLE
Repo: Updated URLs of sites that support HTTPS.

### PR DESCRIPTION
**Pull request by EricDunsworth**
This pretty much covers all of this repo's URLs, but excludes the following:
- Sites that lack HTTPS support
- Sites with broken HTTPS support (e.g. site looks broken when accessed via HTTPS or has an invalid HTTPS certificate)
- Sites that no longer exist
- URLs that resolve to different pages depending on how they're accessed (HTTP or HTTPS)
- Doctype declarations
- XMLNS attributes (http:-prefixed values are literally the names of the namespaces)
- Namespaces in htmllint task's ignore rules
- vocab="http://schema.org/" attributes (was asked to stick with HTTP for them in wet-boew/wet-boew#8674)
- @prefix and @context values in the search template's schema/shape graph code blocks

Port of wet-boew/wet-boew#8674.

### CODE REVIEW

File: site/pages/ajax/sitemenu-v5-en.hbs [hunk 1]
```diff
@@ -12,8 +12,8 @@ language: "en"
 		<li role="separator"></li>
 		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/jobs/opportunities.html">Find a job</a></li>
 		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/jobs/training.html">Training</a></li>
-		<li role="presentation"><a role="menuitem" tabindex="-1" href="http://www.canada.ca/business-management">Hire and manage employees</a></li>
-		<li role="presentation"><a role="menuitem" tabindex="-1" href="http://www.canada.ca/start-business">Starting a business</a></li>
+		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/business-management">Hire and manage employees</a></li>
+		<li role="presentation"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/start-business">Starting a business</a></li>
```
Comment (duboisp) [inline site/pages/ajax/sitemenu-v5-en.hbs]: We will need to do apply the same update on our WMS hosted and CDN version of the menu.

/cc @thangle2013 @therealolivergross @GormFrank

File: site/pages/gcweb-theme/v5-migration.hbs [hunk 2]
```diff
@@ -347,7 +347,7 @@
...
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/policing.html"&gt;Policing, justice and emergencies&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/transport.html"&gt;Transport and infrastructure&lt;/a&gt;&lt;/li&gt;
-			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="http://international.gc.ca/world-monde/index.aspx?lang=eng"&gt;Canada and the world&lt;/a&gt;&lt;/li&gt;
+			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://international.gc.ca/world-monde/index.aspx?lang=eng"&gt;Canada and the world&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html"&gt;Money and finances&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html"&gt;Science and innovation&lt;/a&gt;&lt;/li&gt;
```
Comment (duboisp) [inline site/pages/gcweb-theme/v5-migration.hbs]: Just remember, that line (the link) need to be udpated with the latest code. It should point to "science" and not "finance".

/cc @GormFrank

File: site/pages/ministerial-en.hbs [hunk 3]
```diff
@@ -65,7 +65,7 @@
 				</div>
 				<div class="wb-feeds limit-3 col-md-4">
 					<ul class="feeds-cont list-unstyled lst-spcd">
-						<li> <a href="http://news.gc.ca/web/fd-en.do?mthd=dpt&amp;ft=atom&amp;crtr.dpt1D=6675" rel="external">Government of Canada News Releases</a> </li>
+						<li> <a href="https://news.gc.ca/web/fd-en.do?mthd=dpt&amp;ft=atom&amp;crtr.dpt1D=6675" rel="external">Government of Canada News Releases</a> </li>
```
Comment (duboisp) [inline site/pages/ministerial-en.hbs]: I think we have a new link to the news feed. All template would need to be updated.

/cc @GormFrank

File: site/pages/ajax/sitemenu-v5-en.hbs (hunk 1 above)
Comment (GormFrank) [reply site/pages/ajax/sitemenu-v5-en.hbs]: This has been done in a recent JIRA ticket

File: site/pages/gcweb-theme/v5-migration.hbs (hunk 2 above)
Comment (GormFrank) [reply site/pages/gcweb-theme/v5-migration.hbs]: Oh so that menu should be up-to-date too, I see.

File: site/pages/ministerial-en.hbs (hunk 3 above)
Comment (GormFrank) [reply site/pages/ministerial-en.hbs]: I am not sure if you are talking about a specific link, but there is indeed a "new" way to link to a department's feed through the advanced search, here is an example that brings you to all news from Foreign affairs and development: https://www.canada.ca/en/news/advanced-news-search/news-results.html?dprtmnt=departmentofforeignaffairstradeanddevelopment

**Comment by EricDunsworth** [comment]
FYI I've fixed the merge conflicts related to #1576.

Didn't change anything related to the line comments since seem to be out of scope.

The conversation I marked as a resolved ties into #1589. Not sure if the leftover conversations have been fully resolved yet.

**Comment by duboisp** [comment]
Pre-approved at the technical review. The news link are going to be reverted

**Comment by EricDunsworth** [comment]
@GormFrank FYI I've just updated this PR. I reverted all news.gc.ca links back to HTTP since that site is currently timing out forever when connecting to it via HTTPS.

**Pull request merged**